runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsConfig.java

```diff
@@ -59,6 +59,17 @@ default boolean pointsMessage()
 
 	@ConfigItem(
 		position = 2,
+		keyName = "repositionPoints",
+		name = "Move points box to the right",
+		description = "Reposition the raids points overlay to the right"
+	)
+	default boolean repositionPoints()
+	{
+		return true;
```

Comment:
Why is this an option? Wouldn't it be best to always position it smartly?

Reply:
probably yeah, I'll just make it so it always does that then